Library/Formula/fpc.rb

```diff
@@ -28,11 +28,15 @@ def install
 
     # Prevent non-executable audit warning
     rm_f Dir[bin/"*.rsj"]
+
+    # Generate a default fpc.cfg to set up unit search paths
+    system "#{bin}/fpcmkcfg", "-p", "-d", "basepath=#{lib}/fpc/#{version}", "-o", "#{etc}/fpc.cfg"
   end
 
   test do
     hello = <<-EOS.undent
       program Hello;
+      uses GL;
```

Comment:
i consider that a feature, it *needs* to be destroyed so that new defaults can be installed
any user configuring custom defaults can just add a fpc.cfg to their home directory